app/views/publishers/home.html.slim

```diff
@@ -23,6 +23,14 @@ script type="text/html" id="confirm_default_currency_modal_wrapper"
 
 
 = hidden_field_tag 'publisher_id', current_publisher.id
+- if current_publisher.promo_registrations.present?
+  = hidden_field_tag 'all-channels-referral-stats-starting-month', current_publisher.promo_registrations.order(created_at: :asc).first.created_at.strftime("%Y-%m-%d")
+  - referrals_hidden_tags = []
+  - current_publisher.promo_registrations.each do |promo_registration|
+    - hidden_tag_name = 'referral-stat-' + promo_registration.referral_code
```

Comment:
small fix, let's adjust from the all caps to something like "Stats for referral code: BATS123" rather than "REFERRAL-STAT - BATS123"